doc/bibliography.html

```diff
@@ -438,6 +438,16 @@ <H2>Bibliography</H2>
 <BR><em>Generating random spanning trees more quickly than the cover time</em>.
 ACM Symposium on the Theory of Computing, pp. 296-303, 1996.
 
+<p></p><dt><a name="edmonds65:_max_weighted_match">74</a>
+<dd>J. Edmonds<br>
+<em>Maximum Matching and a Polyhedron with 0, 1-Vertices</em><br>
+Journal of Research of the National Bureau of Standards B 69, pp. 125-130, 1965.
+
+<p></p><dt><a name="gabow90">75</a>
+<dd>Harold N. Gabow<br>
+<em>Data Structures for Weighted Matching and Nearest Common Ancestors with Linking</em><br>
+Proceedings of the First Annual ACM-SIAM Symposium on Discrete Algorithms, pp. 434-443, 1990.
+
```

Comment:
I know this publication from 1990 is more recent, but maybe his original paper on the implementation is more relevant and specific? https://dl.acm.org/citation.cfm?id=321942

Reply:
I agreed, change will be made. 
Just to be clear, the labeling method is not generalized to weighted matching in the paper you mentioned, though it does can be. 
I have to admit that the 1990 paper focused more about improving the time complexity from O(n^3) to O(n(m+nlogn)) using link-cut-trees to find nearest common ancestor, which is not implemented in this PR..

Reply:
I'm not quite sure what you mean about the labeling method is not generalized in that paper?

But otherwise, glad my suggestion was helpful.

I didn't realize that NCA (which I think is typically called LCA?) is part of this algorithm, although I haven't finished going through it yet. Where is it in here?

Reply:
I mean, the labeling procedure in the paper you mentioned only applies to unweighted matching. The author just claimed at the end that it could be generalized to weighted matching.

Link-cut-tree based NCA/LCA is not implemented in this PR. I am using a trivial method, which is used in `max_cardinality_matching.hpp` as well.

In fact, this 1990 citation should not be deleted... I didn’t look carefully where it is cited when replying you. It’s not cited for algorithm reference, but for pointing out the best known time complexity is not O(n^3).
I cited Edmond: “Maximum Matching and a Polyhedron with 0, 1-Vertices” for my implementation. 
Of course I will still add the paper you mentioned, because Edmond’s paper did not elaborate implementation detail for labeling procedure.